folly/Conv.h

```diff
@@ -754,11 +754,12 @@ typename std::enable_if<
 toAppend(
```

Comment:
This changes a public API in an open-source project. You can't do this. You can add the flags as the last argument with a default value of `NO_FLAGS`, or add another overload of `toAppend` with the old interface. Please upstream this change to `facebook/folly`.